File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x < 0) {
            return false;
        }
        int rev = 0;
        int temp = x;
        while(temp!=0) {
            int rem = temp%10;
            rev = rev*10 + rem;
            temp = temp/10;
        }
        if(rev==x) {
            return true;
        } else {
            return false;
        }
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0 :
            return False
        
        rev = 0
        temp = x
        while x>0:
            rem = x%10
            rev = rev*10+rem
            x = x//10

        print(rev)
        if temp==rev:
            return True
        else:
            return False
            